code/workspace/architecting-templates/hosted-zone.template

```diff
@@ -0,0 +1,79 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
[PlaceHolder] Please duplicate any changes from solution template here...

Reply:
Changes added in a new commit, thanks!